mscore/preferences.cpp

```diff
@@ -1154,13 +1154,13 @@ void PreferenceDialog::resetShortcutClicked()
     
 void PreferenceDialog::saveShortcutListClicked()
       {
-      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)");
+      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)", 0, preferences.nativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog);
```

Comment:
The native dialog option you mean? It makes a huge difference if it's true or false. This is with false on macosx

<img width="668" alt="capture d ecran 2017-06-22 14 15 49" src="https://user-images.githubusercontent.com/234271/27433519-67ddc802-5755-11e7-8f68-a6adf47da3d4.png">

On mac and windows, the native dialog is a lot faster. On linux, it seems people prefers the Qt one and not the native one. Also we can customize it, adding the "Scores" folder in the right hand list.